extensions/Needinfo/template/en/default/bug/needinfo.html.tmpl

```diff
@@ -202,24 +189,35 @@ $(function() {
         <select name="needinfo_role" id="needinfo_role">
           <option value="other">other</option>
           [% IF NOT bug.reporter.needinfo_blocked %]
-            <option value="reporter">reporter</option>
+            <option
+                data-identity="[% bug.reporter.name_or_login FILTER html %]"
+                value="reporter">reporter</option>
           [% END %]
           [% IF NOT bug.assigned_to.needinfo_blocked %]
-            <option value="assigned_to">assignee</option>
+            <option
+                data-identity="[% bug.assigned_to.name_or_login FILTER html %]"
+                value="assigned_to">assignee</option>
           [% END %]
           [% IF Param('useqacontact') && bug.qa_contact.login != "" && !bug.qa_contact.needinfo_blocked %]
-            <option value="qa_contact">qa contact</option>
+            <option
+                data-identity="[% bug.qa_contact.name_or_login FILTER html %]"
+                value="qa_contact">qa contact</option>
           [% END %]
           [% IF !is_redirect && NOT user.needinfo_blocked %]
-            <option value="user">myself</option>
+            <option
+                data-identity="[% user.name_or_login FILTER html %]"
+                value="user">myself</option>
           [% END %]
           [% FOREACH mentor = available_mentors %]
-            <option [% IF available_mentors.size > 1 %]title="mentor"[% END %] value="[% mentor.login FILTER html %]">
-              [% available_mentors.size == 1 ? "mentor" : mentor.login FILTER html %]
+            <option data-identity="[% mentor.name_or_login FILTER html %][% IF available_mentors.size > 1 %] (mentor)[% END %]"
+                [% IF available_mentors.size > 1 %]title="mentor"[% END %] value="[% mentor.login FILTER html %]">
+                [% available_mentors.size == 1 ? "mentor" : mentor.login FILTER html %]
```

Comment:
`title` is just a tooltip and it actually won't be displayed on `<option>`s. The label is the text within the `<option>` like `<option value="[VALUE]">[LABEL]</option>`.